pkg/pillar/scripts/device-steps.sh

```diff
@@ -191,13 +191,13 @@ fi
 
 P3=$(zboot partdev P3)
 P3_FS_TYPE=$(blkid "$P3"| awk '{print $3}' | sed 's/TYPE=//' | sed 's/"//g')
-if [ -c $TPM_DEVICE_PATH ] && ! [ -f $CONFIGDIR/disable-tpm ] && [ "$P3_FS_TYPE" = "ext4" ]; then
+if [ -c $TPM_DEVICE_PATH ] && [ "$P3_FS_TYPE" = "ext4" ]; then
+    #It is a device with TPM, and formatted with ext4, go ahead with fscrypt
     #Initialize fscrypt algorithm, hash length etc.
     $BINDIR/vaultmgr -c "$CURPART" setupVaults
 
-    #tpm_in_use might have been wiped out during mkfs above.
-    touch $PERSISTCONFIGDIR/tpm_in_use
-    sync
+    #Migrate old installations to new location
```

Comment:
Please check if the file exists in the old location, and log a message about moving it to the new in that case.

Reply:
Use echo "$(date -Ins -u) ..." when logging

Reply:
Done